packages/mobile/src/backup/utils.ts

```diff
@@ -162,3 +170,14 @@ export function isValidBackupPhrase(phrase: string) {
 export function isValidSocialBackupPhrase(phrase: string) {
   return isValidMnemonic(phrase, 13)
 }
+
+export async function encryptMnemonic(phrase: string) {
+  const pepper = await retrieveOrGeneratePepper()
```

Comment:
I thought we were going to use the pin to encrypt the mnemonic in case the pepper is lost before the user does the backup flow?

Reply:
Using password instead.